### PR TITLE
Fix #77

### LINKED ISSUE #77: /linodes list view does not play nicely with groups

### PR DESCRIPTION
**Pull request by SirCmpwn**
Makes grouped Linodes look better on the grid and list views.

### CODE REVIEW

File: scss/linodes/linodes.scss [hunk 1]
```diff
@@ -1,5 +1,21 @@
 @import 'create';
 
+.linodes-page {
+    h2 {
+        font-size: 1.2rem;
```
Comment (eatonphil) [inline scss/linodes/linodes.scss]: -1

Comment (SirCmpwn) [inline scss/linodes/linodes.scss]: I thought we were using rems for font sizes now?

Comment (eatonphil) [inline scss/linodes/linodes.scss]: Yes but this is already defined somewhere else. We shouldn't overwrite it anywhere. If you wanted that size you should use h3.

Comment (SirCmpwn) [inline scss/linodes/linodes.scss]: Wrong. We should be using the semantically appropriate headers and tweak them with styles if necessary. And this makes it bigger, not smaller. h2s are oddly small by default.

Comment (eatonphil) [inline scss/linodes/linodes.scss]: There's no reason this has to be h2 semantically.

Comment (SirCmpwn) [inline scss/linodes/linodes.scss]: Yeah there is. It's the second level header on the page. An h2.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**